[2025-09-21T11:39:05.364+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:38:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:39:05.365+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:38:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:38:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:39:05.438+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:38:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:39:05.440+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:38:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:38:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:39:05.500+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:38:00+00:00 [queued]>
[2025-09-21T11:39:05.511+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:38:00+00:00 [queued]>
[2025-09-21T11:39:05.512+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-09-21T11:39:05.531+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): trigger_model_training> on 2025-09-21 11:38:00+00:00
[2025-09-21T11:39:05.542+0000] {standard_task_runner.py:57} INFO - Started process 1243 to run task
[2025-09-21T11:39:05.546+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'trigger_model_training', 'scheduled__2025-09-21T11:38:00+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmpua18p62u']
[2025-09-21T11:39:05.550+0000] {standard_task_runner.py:85} INFO - Job 108: Subtask trigger_model_training
[2025-09-21T11:39:05.616+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:38:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:39:05.616+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:38:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:38:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:39:05.618+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:38:00+00:00 [running]> on host 36da5371e73d
[2025-09-21T11:39:05.744+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='trigger_model_training' AIRFLOW_CTX_EXECUTION_DATE='2025-09-21T11:38:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-21T11:38:00+00:00'
[2025-09-21T11:39:10.452+0000] {python.py:194} INFO - Done. Returned value was: 2f0f2500874141e193abf0d586baa30e
[2025-09-21T11:39:10.520+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=trigger_model_training, execution_date=20250921T113800, start_date=20250921T113905, end_date=20250921T113910
[2025-09-21T11:39:10.584+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-21T11:39:10.637+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
